assessment.md

```diff
@@ -269,39 +269,75 @@ TODO: add description for each assessment level for this skill.
+  * :neutral_face: = Student interacts with the group moderately and is in proper communication.
+  * :blush: = Student helps other group members and interacts frequently with the group.
+  * :laughing: = Student leads the group discussions and helps other students.
 
 * Participation
 
-TODO: add description for each assessment level for this skill.
+  * :flushed: = Student fails to do their part in the group. They show no work ethic or communication skills.
+  * :worried: = Student fails to entirely complete their part in the group. They lack work ethic and communication skills.
+  * :neutral_face: = Student does thier part in the group, but lacks work ethic and commincation skills.
+  * :blush: = Student does thier part in the group, but does not show entirely effective work eithic and communication skills.
+  * :laughing: = Student does thier part in the group. They work effectively and efficiently with group members while maintaining good communication.
 
 * Conduct
 
-TODO: add description for each assessment level for this skill.
+  * :flushed: = Student does not act in accordance with the code of conduct. They are a burden for the group.
+  * :worried: = Student does not entirely act in accordance with the code of conduct. They hinder the group with their behavior.
+  * :neutral_face: = Student acts in accordance with the code of conduct, but they do not anything to the group.
+  * :blush: = Student acts in accordance with the code of conduct. They are kind and respectful, but not the most helpful.
+  * :laughing: = Student acts in accordance with the code of conduct. They are also kind, respectful, open-minded and eager to learn.
 
 * Decisions
 
-TODO: add description for each assessment level for this skill.
+  * :flushed: = Student makes decisions without consulting the rest of the group.
+  * :worried: = Student consults the group, but still makes their own decisions.
+  * :neutral_face: = Student consults the group to come to a consensus of what decision should be made.
+  * :blush: = Student sonsults the group to come to a consensus of what decision should be made, whether or not they believe it to be the best answer
+  * :laughing: = Student consults the group and contributes to a healthy conversation to come to a consensus of what decision should be made, whether or not they believe it to be the best answer, and supports it.
 
 * Postmortems
 
-TODO: add description for each assessment level for this skill.
+  * :flushed: = Student does not contribute when fixing remaining issue and blames other group members.
+  * :worried: = Student is unwilling to fix postmortems in a proper manner, and isn't helping the team.
+  * :neutral_face: = Student subtly helps the group fix remaining issues.
+  * :blush: = Student helps the group fix multiple issues, and contributes a lot.
+  * :laughing: = Student leads the group, and guides everyone to fix the issues as a team.
```

Comment:
The use of "postmortems" and "fix remaining issues" in this section is unclear and quite limited. Postmortem is sort of a review session where the group comes together and figures out what went well and what didn't so that the group can mitigate such failures in the future. The format of postmortem also goes far beyond just a "meeting". I think the group should look closer into this term and come up with clearer assessments.